components/link/link-styles.js

```diff
@@ -0,0 +1,23 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+
+export const linkStyles = css`
+	.d2l-link, .d2l-link:visited, .d2l-link:active, .d2l-link:link {
```

Comment:
I'm slightly worried about using the `d2l-link` CSS class since that's what the LMS uses in BSI. Just testing that out now.

Reply:
Seems to be OK.